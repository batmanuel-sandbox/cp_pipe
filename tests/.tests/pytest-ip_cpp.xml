<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="0" name="pytest" skips="0" tests="5" time="2.994"><testcase classname="ip_cpp.tests.test_ip_cpp.IpCppTestCase" file="tests/test_ip_cpp.py" line="51" name="testClassInstantiation" time="2.4110422134399414"></testcase><testcase classname="ip_cpp.tests.test_ip_cpp.IpCppTestCase" file="tests/test_ip_cpp.py" line="44" name="testExample" time="0.0004680156707763672"></testcase><testcase classname="ip_cpp.tests.test_ip_cpp.IpCppTestCase" file="tests/test_ip_cpp.py" line="47" name="testImport" time="0.00045800209045410156"></testcase><testcase classname="ip_cpp.tests.test_ip_cpp.TestMemory" file="../../../../ssd/lsstsw/stack3_20171021/stack/miniconda3-4.3.21-10a4fa6/Linux64/utils/14.0-4-gac01bb5+2/python/lsst/utils/tests.py" line="162" name="testFileDescriptorLeaks" time="0.04016256332397461"></testcase><testcase classname="ip_cpp.tests.test_ip_cpp.TestMemory" file="../../../../ssd/lsstsw/stack3_20171021/stack/miniconda3-4.3.21-10a4fa6/Linux64/utils/14.0-4-gac01bb5+2/python/lsst/utils/tests.py" line="142" name="testLeaks" time="0.03344893455505371"></testcase></testsuite>