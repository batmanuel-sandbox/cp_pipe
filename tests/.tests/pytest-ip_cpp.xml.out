==================================================== test session starts =====================================================
platform linux -- Python 3.6.2, pytest-3.2.0, py-1.4.34, pluggy-0.4.0
rootdir: /home/mfl/lsst/ip_cpp, inifile:
plugins: session2file-0.1.9, forked-0.2, xdist-1.20.1, flake8-0.8.1

collecting 0 items
collecting 3 items
collecting 5 items
collecting 5 items

collected 5 items 
run-last-failure: run all (no recorded failures)

tests/test_ip_cpp.py .....

-------------------------- generated xml file: /home/mfl/lsst/ip_cpp/tests/.tests/pytest-ip_cpp.xml --------------------------
====================================================== warnings summary ======================================================
tests/test_ip_cpp.py::IpCppTestCase::testClassInstantiation
  /ssd/lsstsw/stack3_20171021/python/miniconda3-4.3.21/lib/python3.6/site-packages/astropy/extern/bundled/six.py:60: ResourceWarning: unclosed file <_io.TextIOWrapper name='/ssd/lsstsw/stack3_20171021/python/miniconda3-4.3.21/lib/python3.6/site-packages/astropy/extern/bundled/six.py' mode='r' encoding='utf-8'>
    class X(object):
  /ssd/lsstsw/stack3_20171021/python/miniconda3-4.3.21/lib/python3.6/site-packages/astropy/config/configuration.py:541: ConfigurationMissingWarning: Configuration defaults will be used due to OSError:Could not find unix home directory to search for astropy config dir on None
    warn(ConfigurationMissingWarning(msg))
  /ssd/lsstsw/stack3_20171021/python/current/lib/python3.6/importlib/_bootstrap.py:205: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
    return f(*args, **kwds)
  /ssd/lsstsw/stack3_20171021/python/current/lib/python3.6/importlib/_bootstrap.py:205: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
    return f(*args, **kwds)

-- Docs: http://doc.pytest.org/en/latest/warnings.html
============================================ 5 passed, 4 warnings in 2.99 seconds ============================================
